### ISSUE TITLE
PurgeCSS with full UiKit

### ISSUE THREAD

**Issue by bronze** [TomS-/UIKit-TailwindCSS#1]
Hi there.

Found your project on [your forum post](https://processwire.com/talk/topic/22170-my-web-manifesto-uikit-tailwindcss-grid/) while looking to optimize PurgeCSS with UiKit.

Testing it, i noticed you really only import Compronents from UiKit, and if i change the import to the suggested `@import "../node_modules/uikit/src/less/uikit";` the css file blows up in size (to 260kb) with a lot of unused classes (just like im trying to solve).
Did you opt to use Tailwind for personal preference? Have you tried purging UiKit's bloat with PurgeCSS?

thx!

**Comment by TomS-** [comment]
Bronze, 

Sorry for the REALLY late reply. I'm not actually maintaining this, but just thought I'd show people it's possible. The reason I opted to import components on a component-by-component basis is for filesize, you can absolutely purge UIKit CSS, make sure you whitelist the animation classes otherwise some things will go a little strange. Purging UKit will get to to around 50kb, I opt to use Tailwind just because I prefer utility-first and non opinionated frameworks.

**Comment by bronze** [comment]
thanks for taking the time to reply!

I did try to purge UiKit but can't seem to ever get it under 200kb. Tried messing with the regex rules (adding the `@` it uses so much) but I end up with a lot of junk. Figured maybe that's why you only imported some of it.  
If I switch all your UiKit component imports to `@import "../node_modules/uikit/src/less/uikit";` it goes from 23kb to 240kb...  I figured it was the regex but if you saw it going to 50kb then I have to take a step back on this one...

cheers!
BTW, you reminded me of [Donvercoder9](https://xkcd.com/979/) with that msg :)
